### DISCUSSION TITLE
No runner matching the specified labels was found: self-hosted

### DISCUSSION THREAD

**Discussion by dch** [ChristopherHX/github-act-runner#40] [Q&A] [answered]
I'm writing some notes to provide to new users, to include in the FreeBSD ports tree, of this amazing project, to get people started quickly.

- I have pushed a simple `build.yml` https://git.io/J4AuX
- I can register an agent successfully on both arm64 & amd64 FreeBSD:

```
$ github-act-runner configure --unattended \
  --name wintermute-2074 \
  --url https://github.com/:org \
  --token nomNOMnom
$ github-act-runner run
Listening for Jobs
...
```

but when I push or do a PR, the action fails to find a suitable label, even if I add it specifically during the configure step. What am I missing here?

**Comment by ChristopherHX** [comment, marked as answer]
I assume you registered it as an org runner and haven't used any self-hosted runners in that org for public repositories before.

Do you have allowed the default runner group for public repositories? GitHub has it disabled by default.
Please go to `https://github.com/organizations/:org/settings/actions/runners`,
![image](https://user-images.githubusercontent.com/44845461/127209360-cadff387-ebca-45fc-8b37-da3634997c8c.png)
Press the three dots of the default group and change repository access to include public repositories (this implies a potential security risk, due to PR's)
If your repository is private it should work by default for all org repositories.

**Reply by dch** [reply]
that's it - perfect :1st_place_medal: :green_apple: :clap:

**Reply by fish895623** [reply]
This is awesome!! Thanks for answer
